stable/mlrun/values.yaml

```diff
@@ -242,15 +242,15 @@ api:
 
   sidecars:
     logCollector:
-      enabled: false
+      enabled: true
```

Comment:
it will take time to graduate. r u sure you want it to be true by now? perhaps do the fixes and still keep it disabled. then, once GA we will turn it on